packages/postgres-database/src/simcore_postgres_database/models/products.py

```diff
@@ -146,6 +146,7 @@ class ProductLoginSettingsDict(TypedDict, total=False):
         nullable=False,
         doc="Regular expression that matches product hostname from an url string",
     ),
+    # EMAILS --------------------
```

Comment:
always wondering about these comments. not sure I see the point and how sustainable they are.